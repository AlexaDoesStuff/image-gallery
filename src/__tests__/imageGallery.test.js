/* imageGallery.test.js
*
* Example test file for components. Stored in __test__ folder
* Uses react-testing-library, enzyme, jest
*/

import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import ImageGallery from '../components/imageGallery.js';

import { act } from "react-dom/test-utils";
import { mount } from 'enzyme';

Enzyme.configure({ adapter: new Adapter() })

describe('Check if renders correctly on init', () => {
  it('should render', () => {
    const component = shallow(<ImageGallery />)
    expect(component).toMatchSnapshot();
  })
})