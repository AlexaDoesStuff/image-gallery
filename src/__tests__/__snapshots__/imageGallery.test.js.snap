// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if renders correctly on init should render 1`] = `ShallowWrapper {}`;

exports[`Check if renders correctly should render 1`] = `ShallowWrapper {}`;

exports[`Check if renders photo array in Image Gallery should render button 1`] = `ShallowWrapper {}`;

exports[`Check if with json, nodisplay and totals are set #nodisplay and #total should have components 1`] = `ShallowWrapper {}`;
